packages/perseus/src/components/text-input.jsx

```diff
@@ -1,33 +1,38 @@
 /* eslint-disable react/sort-comp */
 // @flow
-import PropTypes from "prop-types";
+import {TextField} from "@khanacademy/wonder-blocks-form";
 import * as React from "react";
 import ReactDOM from "react-dom";
 
-class TextInput extends React.Component<$FlowFixMe> {
-    static propTypes = {
-        value: PropTypes.string,
-        onChange: PropTypes.func.isRequired,
-        className: PropTypes.string,
-        labelText: PropTypes.string,
-        onFocus: PropTypes.func,
-        onBlur: PropTypes.func,
-        disabled: PropTypes.bool,
-    };
-
+type Props = {|
+    value: string,
+    onChange: ($FlowFixMe) => {},
+    className: string,
+    labelText: string,
+    onFocus: () => {},
+    onBlur: () => {},
+    disabled: boolean,
+    id: string,
+|};
+class TextInput extends React.Component<Props> {
     static defaultProps: $FlowFixMe = {
         value: "",
         disabled: false,
     };
```

Comment:
I think we could get flow types for this also.  I think it's just:
```js
type DefaultProps = {|
    value: Props["value"],
    disabled: Props["disabled"],
|}
```

Reply:
Easy! DONE.